bin/varnishtest/vtc.c

```diff
@@ -315,7 +315,7 @@ parse_string(struct vtclog *vl, void *priv, const char *spec)
 	struct vsb *token_exp;
 	char *e, *p, *q, *f, *buf;
 	int nest_brace;
-	int tn;
+	int tn, ln = 0;
```

Comment:
Doesn't `ln` need to start from the spec line number? I'd be surprised if it worked with anything but top comments.

Reply:
ah yes, indeed, my tests appeared to work because because I mishandled whitespaces in a convincing way